### PR TITLE
Fixes #783, remove Test Pilot branding and metrics

### LINKED ISSUE #783: Remove Test Pilot references from website
We also need to remove references to Test Pilot from the website: 

Test Pilot - Color

**Upper left corner:** 
- Remove "A Firefox Test Pilot experiment"
- Center "Firefox Color" with icon 

**Footer:**
- Remove "About Test Pilot" link
- Change link target of Twitter icon to https://twitter.com/firefox/

### PR DESCRIPTION
**Pull request by mgillespy**

### CODE REVIEW

File: src/web/lib/components/AppHeader/index.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@
   z-index: 2;
 
   .app-header__content {
-    align-items: center;
+    justify-content: center;
```
Comment (lmorchard) [inline src/web/lib/components/AppHeader/index.scss]: Nit: Is this an intentional design change? I don't want to second guess it, if so. Just looks different to me.

Comment (mgillespy) [reply src/web/lib/components/AppHeader/index.scss]: Yes, @caitmuenster suggested we center the 'Firefox Color' logo & text after removing the Test Pilot text from the header.

Comment (lmorchard) [reply src/web/lib/components/AppHeader/index.scss]: All good then!

Comment (mgillespy) [reply src/web/lib/components/AppHeader/index.scss]: Thanks, @lmorchard ! :)

**Pull request merged**